php-frontend/src/main/java/org/sonar/plugins/php/api/visitors/PHPVisitorCheck.java

```diff
@@ -572,4 +574,12 @@ protected void scan(Tree tree) {
     visitCompilationUnit(context.tree());
     return context().getIssues();
   }
+
+  protected final QualifiedName getFullyQualifiedName(NamespaceNameTree namespaceNameTree) {
```

Comment:
This method should also be tested in "php-frontend" to please the coverage and not only in "php-checks".